tests/app/services/test_query_builder.py

```diff
@@ -206,46 +223,26 @@ def test_highlight_block_sets_encoder_to_html():
     assert_equal(query["highlight"]["encoder"], "html")
 
 
-def test_highlight_block_contains_correct_fields():
+@pytest.mark.parametrize('example, expected', (
+    ("id", True),
+    ("lot", True),
+    ("serviceName", True),
+    ("serviceSummary", True),
+    ("serviceFeatures", True),
+    ("serviceBenefits", True),
+    ("serviceTypes", True),
+    ("supplierName", True)
+))
```

Comment:
Can we add some False assertions to try to validate that some blocks aren't included?